<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_15">
    <item>
     <widget class="QGroupBox" name="groupBox_tableData">
      <property name="title">
       <string>表格数据</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <widget class="QGroupBox" name="groupBox_sort">
           <property name="title">
            <string>排序</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>排序字段</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_sortItem"/>
              </item>
              <item>
               <widget class="QRadioButton" name="radioButton_ascend">
                <property name="text">
                 <string>升序</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioButton_Descend">
                <property name="text">
                 <string>降序</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_dataFilter">
           <property name="title">
            <string>数据过滤</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_13">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_12">
              <item>
               <widget class="QRadioButton" name="radioButton_male">
                <property name="text">
                 <string>男</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioButton_female">
                <property name="text">
                 <string>女</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioButton_all">
                <property name="text">
                 <string>全显示</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="tableView"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_curRecord">
      <property name="title">
       <string>当前记录</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,1">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="1,1">
           <item>
            <widget class="QGroupBox" name="groupBox_5">
             <property name="title">
              <string/>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>工号</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spinBox_numberID">
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="maximum">
                   <number>9999</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="3,3">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>姓名</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_name"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,1">
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>性别</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_gender"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,1">
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>出生日期</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_birth"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,1">
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>省份</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_province"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,1">
                <item>
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>部门</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_department"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <item>
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>工资</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spinBox_salary">
                  <property name="minimum">
                   <number>1000</number>
                  </property>
                  <property name="maximum">
                   <number>999999</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_6">
             <property name="title">
              <string>照片</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLabel" name="label_photo">
                <property name="text">
                 <string>TextLabel</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_notation">
           <property name="title">
            <string>备注</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPlainTextEdit" name="plainTextEdit"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>875</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_open"/>
   <addaction name="separator"/>
   <addaction name="action_add"/>
   <addaction name="action_insert"/>
   <addaction name="action_delete"/>
   <addaction name="separator"/>
   <addaction name="action_setPhoto"/>
   <addaction name="action_delPhoto"/>
   <addaction name="action_scan"/>
   <addaction name="separator"/>
   <addaction name="action_submit"/>
   <addaction name="action_Revert"/>
   <addaction name="separator"/>
   <addaction name="action_quit"/>
  </widget>
  <action name="action_open">
   <property name="text">
    <string>打开</string>
   </property>
   <property name="toolTip">
    <string>打开数据库</string>
   </property>
  </action>
  <action name="action_add">
   <property name="text">
    <string>添加</string>
   </property>
   <property name="toolTip">
    <string>添加项</string>
   </property>
  </action>
  <action name="action_insert">
   <property name="text">
    <string>插入</string>
   </property>
   <property name="toolTip">
    <string>插入项</string>
   </property>
  </action>
  <action name="action_delete">
   <property name="text">
    <string>删除</string>
   </property>
   <property name="toolTip">
    <string>删除项</string>
   </property>
  </action>
  <action name="action_setPhoto">
   <property name="text">
    <string>设置照片</string>
   </property>
   <property name="toolTip">
    <string>清除照片</string>
   </property>
  </action>
  <action name="action_delPhoto">
   <property name="text">
    <string>清除照片</string>
   </property>
   <property name="toolTip">
    <string>清除照片</string>
   </property>
  </action>
  <action name="action_submit">
   <property name="text">
    <string>保存</string>
   </property>
   <property name="toolTip">
    <string>保存修改</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
  <action name="action_Revert">
   <property name="text">
    <string>取消</string>
   </property>
   <property name="toolTip">
    <string>取消修修改</string>
   </property>
  </action>
  <action name="action_scan">
   <property name="text">
    <string>涨工资</string>
   </property>
   <property name="toolTip">
    <string>涨工资</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>action_quit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>437</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
